sandbox-backend/src/main/java/com/exadel/sandbox/team5/entity/EmployeeDiscount.java

```diff
@@ -0,0 +1,24 @@
+package com.exadel.sandbox.team5.entity;
+
+import lombok.*;
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+
+@Entity
+@Table(name = "employee_discount")
+public class EmployeeDiscount extends BaseEntity{
+
+    @Column(name = "employeeId", nullable = false)
+    private long employeeId;
+
+    @Column(name = "discountId", nullable = false)
+    private long discountId;
```

Comment:
It is better to use relationship many to one between `EmployeeDiscount` and `Discount` entities.